app/services/pledge_form_generator.rb

```diff
@@ -0,0 +1,144 @@
+class PledgeFormGenerator
+  attr_reader :patient, :user, :case_manager_name
+
+  def initialize(user, patient, case_manager_name)
+    @patient = patient
+    @user = user
+    @case_manager_name = case_manager_name.to_s
+  end
+
+  def patient_amount
+    @patient.pregnancy.dcaf_soft_pledge
```

Comment:
Nah, the amount in the form is strictly the amount we're shipping -- clinics work with other funds for their portions and we tend not to bundle or have a 'central' fund handle remits for everyone, if that makes sense. 

`to_s`ing it at the controller level is a good call, I'll move that now.